src/main/java/org/dependencytrack/persistence/DefaultObjectGenerator.java

```diff
@@ -238,7 +238,7 @@ private void loadDefaultRepositories() {
             qm.createRepository(RepositoryType.GEM, "rubygems.org", "https://rubygems.org/", true, false);
             qm.createRepository(RepositoryType.HEX, "hex.pm", "https://hex.pm/", true, false);
             qm.createRepository(RepositoryType.MAVEN, "central", "https://repo1.maven.org/maven2/", true, false);
-            qm.createRepository(RepositoryType.MAVEN, "atlassian-public", "https://maven.atlassian.com/content/repositories/atlassian-public/", true, false);
+            qm.createRepository(RepositoryType.MAVEN, "atlassian-public", "https://packages.atlassian.com/content/repositories/atlassian-public/", true, false);
```

Comment:
Would this apply to existing installations? The createRepository function checks whether a repository for this type and identifier already exists. Maybe this should be added to the v450Updater as well.

Reply:
You're correct @AbdelHajou. This change should be made in `v450Updater` as well.

@pmckeown See here for how it was done in v3.8.0 for a change in the Maven Central URL: https://github.com/DependencyTrack/dependency-track/blob/3.8.0/src/main/java/org/dependencytrack/upgrade/v380/v380Updater.java#L41

Reply:
Great point. Hadn’t considered that. Will take at the update for 3.8.0 tomorrow.